### ISSUE TITLE
AttributeError: 'gi.repository.Adw' object has no attribute 'PreferencesDialog'. Did you mean: 'PreferencesWindow'?

### ISSUE THREAD

**Issue by raintonr** [SoongNoonien/plattenalbum#114]
Trying to run on Fedora (FC39) and getting this error:

```
$ plattenalbum 
Traceback (most recent call last):
  File "/usr/local/bin/plattenalbum", line 995, in <module>
    class SettingsDialog(Adw.PreferencesDialog):
                         ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/site-packages/gi/module.py", line 126, in __getattr__
    raise AttributeError("%r object has no attribute %r" % (
AttributeError: 'gi.repository.Adw' object has no attribute 'PreferencesDialog'. Did you mean: 'PreferencesWindow'?
```

**Comment by SoongNoonien** [comment]
Hi!
As indicated in the README, you need at least libadwaita 1.5 to run the latest git version. So either you use v2.0.1 or you can try building with gnome-builder. Building with gnome-builder will install the Gnome 46 runtime which includes libadwaita 1.5.

**Comment by raintonr** [comment]
I did see the dependencies but didn't realise having an old version would manifest itself as such an error. Something more useful like, "You seem to be running an old version of libadwaita" would be nice.

Anyhow, no worries, because FC40 is newly released and includes 1.5 so all good :)
